### PR TITLE
Artakm/bootstrap v3 update

### PR DESCRIPTION
**Pull request by mkArtakMSFT**
Updated IdentityUI V3 to reference the 3.4.0 version of Bootstrap

### CODE REVIEW

File: src/Identity/src/UI/wwwroot/V3/Identity/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.svg [hunk 1]
```diff
@@ -285,4 +285,4 @@
 <glyph unicode="&#x1f511;" d="M250 1200h600q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-150v-500l-255 -178q-19 -9 -32 -1t-13 29v650h-150q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5zM400 1100v-100h300v100h-300z" />
 <glyph unicode="&#x1f6aa;" d="M250 1200h750q39 0 69.5 -40.5t30.5 -84.5v-933l-700 -117v950l600 125h-700v-1000h-100v1025q0 23 15.5 49t34.5 26zM500 525v-100l100 20v100z" />
 </font>
-</defs></svg>
\ No newline at end of file
+</defs></svg>
```
Comment (ryanbrandenburg) [inline src/Identity/src/UI/wwwroot/V3/Identity/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.svg]: Weird that this is the only thing changed in this file.

**Review by ryanbrandenburg — APPROVED** [review]
LGTM. We should update Templating at roughly the same time.

File: src/Identity/src/UI/wwwroot/V3/Identity/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.svg (hunk 1 above)
Comment (mkArtakMSFT) [reply src/Identity/src/UI/wwwroot/V3/Identity/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.svg]: Yeah, we don't own these - so it is what it is. I just overwritten the files.

**Comment by mkArtakMSFT** [comment]
The reported test failures are from unrelated area, so I'm moving ahead with merging this change in.

File: src/Identity/src/UI/THIRD-PARTY-NOTICES.txt [hunk 2]
```diff
@@ -104,11 +104,35 @@ LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
...
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
+License notice for Bootstrap v4.1.3
```
Comment (Eilon) [inline src/Identity/src/UI/THIRD-PARTY-NOTICES.txt]: @mkArtakMSFT - Why do we have two notices now?

**Pull request merged**